packages/pages/src/App.js

```diff
@@ -70,6 +72,28 @@ export default provideState({
       this.state.content = "";
       await this.effects.refreshEntries();
     },
+    async deleteEntry(
+      _,
+      {
+        target: { value },
+      }
+    ) {
+      this.state.id = value;
```

Comment:
Why do you need this?

Reply:
This allows to recover the "value".

Reply:
I don't understand what you mean, you can use `value` directly in the `fetch` call.